### PR TITLE
Task review fixes

### PR DESCRIPTION
**Pull request by codymccoy**

### CODE REVIEW

File: UserControls/UserControls.vbproj [hunk 1]
```diff
@@ -53,13 +53,13 @@
...
     </Reference>
     <Reference Include="fda-model, Version=1.1.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\USACE.HEC.FDA.Model.1.1.0-alpha-2022-07-15-2223\lib\net48\fda-model.dll</HintPath>
+      <HintPath>..\packages\USACE.HEC.FDA.Model.1.1.0-alpha-2022-07-19-1451\lib\net48\fda-model.dll</HintPath>
     </Reference>
     <Reference Include="Model, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
       <HintPath>..\packages\HEC.MVVMFramework.Model.1.0.0-alpha-2022.7.8.2331\lib\netstandard2.0\Model.dll</HintPath>
     </Reference>
     <Reference Include="Statistics, Version=1.1.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\USACE.HEC.FDA.Statistics.1.1.0-alpha-2022-07-15-2223\lib\net48\Statistics.dll</HintPath>
```
Comment (Brennan1994) [inline UserControls/UserControls.vbproj:62]: Go ahead and drop this reference. This reference shouldn't exist at all. User controls doesn't need to know anything about statistics.

File: ViewModel/Alternatives/Results/DamageWithUncertaintyVM.cs [hunk 2]
```diff
@@ -29,12 +29,12 @@ public DamageWithUncertaintyVM(AlternativeResults results, DamageMeasureYear dam
             if (double.IsNaN(discountRate))
             {
                 RateAndPeriodVisible = false;
-                ProbabilityExceedsValueLabel = "Probability that damage exceeds indicated value";
```
Comment (Brennan1994) [inline ViewModel/Alternatives/Results/DamageWithUncertaintyVM.cs:32]: For labels like these you were using a String Constants class for awhile. I've also seen in other people's code having string constants at the top of the class, just so they're in the same class they're used, but still easy to find and change easily. Does it make sense to use one of those strategies here?

Comment (Brennan1994) [reply ViewModel/Alternatives/Results/DamageWithUncertaintyVM.cs:32]: There's a few places you're using string labels in code in this pull request. I'll let this comment refer to  all of them.

**Review by Brennan1994 — CHANGES_REQUESTED** [review]
Mostly looks good. just a couple notes

File: UserControls/UserControls.vbproj (hunk 1 above)
Comment (Brennan1994) [reply UserControls/UserControls.vbproj:62]: also doesn't know anything about model. Drop that guy too.

File: ViewModel/Alternatives/Results/DamageWithUncertaintyVM.cs (hunk 2 above)
Comment (codymccoy) [reply ViewModel/Alternatives/Results/DamageWithUncertaintyVM.cs:32]: I agree, i will add them to the string constants class.

**Pull request merged**